extension/chrome/elements/pgp_pubkey.ts

```diff
@@ -167,8 +184,9 @@ View.run(
       }
     };
 
-    private showKeyNotUsableError = () => {
+    private showKeyNotUsableError = async () => {
       $('.error_container').removeClass('hidden');
+      $('.error_introduce_label').text(`This OpenPGP key is not usable.\n${await this.getErrorText()}`);
```

Comment:
let's use `<small>` html tag for error description, it will take less space than normal text size.
also `error: error encrypting message:` part can't be removed, so error message will begin directly from error reason:
<img width="360" src="https://github.com/user-attachments/assets/cd66edfc-e781-40ea-bf87-aca947ddea6b" />

and incorrect key row is misaligned with other rows, probably because of smaller `view key` button. let's make it the same width as `import key` button.

Reply:
Sounds good!